src/app/Http/Controllers/RoleCrudController.php

```diff
@@ -61,17 +61,15 @@ public function setup()
 
     public function store(StoreRequest $request)
     {
-        //otherwise, changes won't have effect
-        \Cache::forget('spatie.permission.cache');
-
-        return parent::storeCrud();
+        $response= parent::storeCrud();
+        app(PermissionRegistrar::class)->forgetCachedPermissions();
```

Comment:
Hmm... I wonder - didn't ```\Cache::forget('spatie.permission.cache');``` take care of the same thing? Did it stop working - because as far as I know that's the sole reason this line was there...